lib/foreman_maintain/cli/advanced/task_cleanup_command.rb

```diff
@@ -0,0 +1,65 @@
+module ForemanMaintain
+  module Cli
+    class TaskCleanupCommand < Base
+      include Concerns::SystemHelpers
```

Comment:
`SystemHelpers` concern also have `execute` method that's why creating a conflict between these two methods.

Giving below error on `foreman-maintain advanced task-cleanup`:

```
foreman_maintain/lib/foreman_maintain/cli/advanced/task_cleanup_command.rb:22:in `execute': wrong number of arguments (1 for 0) (ArgumentError)
	from /root/foreman_maintain/lib/foreman_maintain/concerns/system_helpers.rb:89:in `find_package'
	from /root/foreman_maintain/lib/foreman_maintain/concerns/system_helpers.rb:100:in `server?'
	from /root/foreman_maintain/lib/foreman_maintain/cli/advanced/task_cleanup_command.rb:27:in `execute'
```